python/lsst/resources/http.py

```diff
@@ -221,6 +221,37 @@ def finalurl(r: requests.Response) -> str:
     return destination_url
 
 
+# Tuple (path, block_size) pointing to the location of a local directory
+# to save temporary files and the block size of the underlying file system
+_TMPDIR = None
+
+
+def _get_temp_dir() -> Tuple[str, int]:
+    """Return the temporary directory path and block size.
+
+    This function caches its results in _TMPDIR.
+    """
+    global _TMPDIR
+    if _TMPDIR:
+        return _TMPDIR
```

Comment:
It might be worth adding a second `as_local()` test to ensure that we get code coverage on this line.